.bazelversion

```diff
@@ -1 +1 @@
-bazelbuild/7.3.1
+8.0.1
```

Comment:
At this point 8.1.0 is out, so let's go to that instead.

Reply:
I upgraded to our custom version, which fixes the multiplex sandboxing bug and is based on v8.1.1.